apps/backend/.env.example

```diff
@@ -18,11 +18,16 @@ DB_DIALECT=pg
 DB_POOL_MIN=2
 DB_POOL_MAX=10
 
+#
+# DURATIONS
+#
+DURATION_PASSWORD_RESET_LINK=30mins
+DURATION_SESSION=24hrs
```

Comment:
```suggestion
PASSWORD_RESET_LINK_DURATION=30mins
SESSION_DURATION=24hrs
```

Reply:
But EnvironmetSchema will remain the same? I think that config.ENV.DURATIONS.[DURATION_NAME] is a better interface

Reply:
Yes, I agree, the naming in EnvironmentSchema looks good